app/Http/Controllers/BoardConfigController.php

```diff
@@ -60,13 +62,13 @@ public function store(Request $request): RedirectResponse
             'columns.*' => 'string|min:1|max:255',
         ]);
 
-        BoardConfig::create([
+        $boardId = DB::table("board_configs")->insertGetId([
             'title'    => $validated['title'],
-            'columns'  => $validated['columns'],
+            'columns'  => json_encode($validated['columns']),
```

Comment:
Let inertia handle this, don't encode by yourself. You're changing core behavior of the boards.